### PR TITLE
Efficient matrix initialization

### PR DESCRIPTION
**Pull request by moussaka-crypto**
Wrote a template method for the x,y,z matrices to replace the repetition, also added for loops to add and multiply methods

### CODE REVIEW

File: src/main/java/com/example/matrixcalc/Controller.java [hunk 1]
```diff
@@ -229,15 +137,14 @@ public void multiply() {
...
+        for (int i = 0; i < 3; i++) {
+            for (int j = 0; j < 3; j++) {
+                String fieldName = "z" + (i + 1) + (j + 1);
+                this.getClass().getDeclaredField(fieldName).set(this, result[i][j]);
+            }
+        }
     }
 }
 
+// Javata basi mastiqta
```
Comment (TadNodir) [inline src/main/java/com/example/matrixcalc/Controller.java:150]: pls remove this line

File: src/main/java/com/example/matrixcalc/Controller.java [hunk 2]
```diff
@@ -107,52 +41,33 @@ void initialize() {
...
+                }
+            }
+        }
     }
 
-    /**
-     * Checks if the given string text is in a number form
-     *
-     * @param text to be checked
-     */
```
Comment (TadNodir) [inline src/main/java/com/example/matrixcalc/Controller.java:145]: Why did you remove documentation?

Comment (moussaka-crypto) [reply src/main/java/com/example/matrixcalc/Controller.java:145]: Because the code is self-explaining
